Runtime/Game/LootLockerSDKManager.cs

```diff
@@ -4281,6 +4281,94 @@ public static void LootLockerPurchaseCatalogItems(string walletId, LootLockerCat
             LootLockerServerRequest.CallAPI(LootLockerEndPoints.purchaseCatalogItem.endPoint, LootLockerEndPoints.purchaseCatalogItem.httpMethod, body, onComplete: (serverResponse) => { LootLockerResponse.Deserialize(onComplete, serverResponse); });
         }
 
+        /// <summary>
+        /// Redeem a purchase that was made successfully towards the Apple App Store for the current player
+        /// </summary>
+        /// <param name="transactionId">The id of the transaction successfully made towards the Apple App Store</param>
+        /// <param name="onComplete">onComplete Action for handling the response</param>
+        public static void LootLockerRedeemAppleAppStorePurchaseForPlayer(string transactionId, Action<LootLockerResponse> onComplete)
+        {
+            if (!CheckInitialized())
+            {
+                onComplete?.Invoke(LootLockerResponseFactory.SDKNotInitializedError<LootLockerResponse>());
+                return;
+            }
+            var body = LootLockerJson.SerializeObject(new LootLockerRedeemAppleAppStorePurchaseForPlayerRequest()
+            {
+                transaction_id = transactionId
+            });
+
+            LootLockerServerRequest.CallAPI(LootLockerEndPoints.redeemAppleAppStorePurchase.endPoint, LootLockerEndPoints.redeemAppleAppStorePurchase.httpMethod, body, onComplete: (serverResponse) => { LootLockerResponse.Deserialize(onComplete, serverResponse); });
+        }
+
+        /// <summary>
+        /// Redeem a purchase that was made successfully towards the Apple App Store for a character that the current player owns
+        /// </summary>
+        /// <param name="transactionId">The id of the transaction successfully made towards the Apple App Store</param>
+        /// <param name="characterId">The id of the character to redeem this transaction for</param>
+        /// <param name="onComplete">onComplete Action for handling the response</param>
+        public static void LootLockerRedeemAppleAppStorePurchaseForPlayer(string transactionId, int characterId, Action<LootLockerResponse> onComplete)
```

Comment:
I would name this `LootLockerRedeemAppleAppStorePurchaseForCharacter`

Reply:
Haha, LOL >.<
Of course, that's correct

Reply:
Fixed